aio/content/marketing/contributors.json

```
@@ -667,6 +667,13 @@
     "groups": ["Angular"],
     "lead": "dennispbrown"
   },
+  "rockument69": {
+    "name": "Tony Bove",
+    "picture": "tonybove.jpg",
```

can you please rename the file to use your username? thank you!

```suggestion
    "picture": "rockument69.jpg",
```

---

OK, included in the following commit.